docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+---
+toc_max_heading_level: 5
+---
+
+# Build
+
+The Unity orb provides the
+[build job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-build) to facilitate the
+process of building your Unity project to multiple platforms.
+
+### Parameters
+
+In this sub-section, you will find information on each parameter the job provides.
+
+#### build-name
+
+- _Default_: `UnityBuild`
+- _Required_: `False`
+- _Type_: `String`
+
+Your build's name. If you choose to compress your artifacts, this name will be used in the
+compressed file.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          build-name: 'MyUnityProject'
+```
+
+#### build-target
+
+- _Default_: `N/A`
+- _Required_: `True`
+- _Type_: `String`
+
+The platform on which you want to run your game. Valid parameters can be found in the Unity Editor
+[documentation](https://docs.unity3d.com/2022.2/Documentation/ScriptReference/BuildTarget.html).
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          build-target: 'StandaloneLinux64'
+```
+
+#### compress
+
+- _Default_: `true`
+- _Required_: `False`
+- _Type_: `Boolean`
+
+Whether to compress the build output to a `.tar.gz` file. Set it to `true` if you want to download
+the build artifacts from the CircleCI dashboard, otherwise, you will have to download each file
+individually. If set to `false` for
+[decompressed WebGL](https://docs.unity3d.com/Manual/webgl-deploying.html) builds, the built project
+can be run directly from the GameCI dashboard.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          compress: true
+```
+
+#### executor
+
+- _Default_: `N/A`
+- _Required_: `True`
+- _Type_: `Executor`
+
+The executor on which you want to run your build job on. Bear in mind that projects using `IL2CPP`
+as the [scripting backend](https://docs.unity3d.com/Manual/scripting-backends.html) should run on
+the same OS as the [build target](#build-target). For example, if you want to build for macOS using
+`IL2CPP`, you must use the
+[macos](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-macos) or
+[macos-runner](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-macos-runner)
+executors.
```

Comment:
```suggestion
The executor that you want to run your build job on. Bear in mind that builds using `IL2CPP` for a 
[scripting backend](https://docs.unity3d.com/Manual/scripting-backends.html) should run on
the same OS as the [build target](#build-target). 

For example, if you want to build for macOS using `IL2CPP`, then you must use the
[macos](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-macos) or
[macos-runner](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-macos-runner)
executor.
```

Reply:
Done ✔️

Reply:
After this change, I reread this section and believe we should use the original. The second sentence is a direct continuation of the first, but making it a paragraph implies otherwise and breaks the reading flow.